docs/b-plus/README.md

```diff
@@ -1,11 +1,9 @@
 # OOPS.. You've got yourself a better Raspberry Pi! It's B+!
 
-It happens to be, that fellows from Raspberry managed to update their Raspberry Pi 3B controller up to Model B+ and we somehow missed it. It means that our book — the one you've found in the box, is somewhat obsolete.
+Good news! Fellows from Raspberry managed to update their Raspberry Pi 3B controller up to Model B+ . It means that our book — the one you've found in the box, is somewhat obsolete, because it was published before the update
```

Comment:
```suggestion
Good news! Fellows from Raspberry Pi Foundation managed to update their Raspberry Pi 3 Model B controller up to Model B+. It means that our book — the one you've found in the box, is somewhat obsolete, because it was published before the update
```